tutorials/labels.md

```diff
@@ -1,38 +1,21 @@
-### Label Layer
+# labels layer tutorial
 
-Another big challenge napari is trying to solve is interactive labeling 
-data. Labels layer enable one to have arbitrary labels on any layer. One 
-can easily superimpose an image and corresponding labels with napari. An 
-example is given below:
+Welcome to the tutorial on **napari** the `Labels` layer!
 
-```python
-from skimage import data
-from skimage.filters import threshold_otsu
-from skimage.segmentation import clear_border
-from skimage.measure import label
-from skimage.morphology import closing, square, remove_small_objects
-import napari
+This tutorial assumes you have already installed **napari**, know how to launch the viewer, and are familiar with its layout. For help with installation see our [installation](installation.md) tutorial. For help getting started with the viewer see our [getting started](getting_started.md) tutorial. For help understanding the organisation of the viewer, including things like the layers list, the layer properties widgets, the layer control panels, and the dimension sliders see our [napari viewer](viewer.md) tutorial.
 
+This tutorial will teach you about the **napari** `Labels` layer, including ...
```

Comment:
:joy: including... NOTHING! YOU GET NOTHING! :joy:

Reply:
:-)